utils/build-presets.ini

```diff
@@ -1472,7 +1473,7 @@ mixin-preset=
 [preset: LLDB_Nested]
 skip-build-benchmarks
 install-destdir=%(swift_install_destdir)s
-llvm-targets-to-build=X86;ARM;AArch64;PowerPC;SystemZ;Mips
+llvm-targets-to-build=X86;ARM;AArch64;PowerPC;SystemZ;Mips;RISCV
```

Comment:
Is there any reason you added it to this lldb preset too? Obviously this is already overriding the Swift defaults, but it is likely only used by the CI and already matches the default, so you could probably just remove this line altogether.

Reply:
I just didn't want to miss anything, I'll remove it.

Reply:
Do you mind just deleting this line too? Might as well clean that up also.

Reply:
I forced pushed and removed all changes in `utils/build-presets.ini`. please check the updated diff.

Reply:
This was added 5 years ago, ab5677b, when the default didn't include Mips, now it does.

Reply:
By "you could probably just remove this line altogether" before, I meant that you could just delete this line from this preset instead of adding to it, a cleanup now that this line matches the default.

Reply:
Sure, I'll do that.

Reply:
Removed. Only Darwin and WASM specify these now in the presets.

Reply:
@fredriss or @ravikandhadai, would you okay removing this line now?